backend/src/main/java/net/vidux/camhub/discovery/RawCameraData.java

```diff
@@ -0,0 +1,13 @@
+package net.vidux.camhub.discovery;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
```

Comment:
A `@Getter` annotációt lecserélném egy `@Value` annotációra, ami nem csak, hogy garantálja, hogy immutable lesz, hanem legenerálja a toString, equals és hashCode metódusokat is.

Reply:
A value nem volt jó a tesztek miatt. Final -t nem lehet mockolni